mvrx-navigation/src/main/kotlin/com/airbnb/mvrx/navigation/MvRxNavDirections.kt

```diff
@@ -21,24 +25,47 @@ import com.airbnb.mvrx.Mavericks
  *      android:label="{mvrx:arg:title}">
  *
  *       <argument
+ *          android:name="mvrx:arg"
+ *          app:argType="com.example.ExampleNavArgs" />
+ *
+ *       <argument
  *          android:name="mvrx:arg:title"
  *          app:argType="string" />
  *
  *   </fragment>
  * ```
  */
-class MvRxNavDirections(
+class MvRxNavDirections private constructor(
     @IdRes private val actionId: Int,
-    private val data: Parcelable,
+    private val parcelable: Parcelable? = null,
+    private val serializable: Serializable? = null,
     private val title: String? = null
 ) : NavDirections {
 
     companion object {
         const val KEY_ARG_TITLE = "mvrx:arg:title"
+        fun create(@IdRes actionId: Int, title: String?, data: Parcelable): MvRxNavDirections =
```

Comment:
You want one of these with just 2 args right and title can default to null for both create functions.

Reply:
Ah, yep.